action.yml

```diff
@@ -48,6 +48,14 @@ runs:
     - name: "Add poetry to PATH"
       run: "ln -s /root/.local/bin/poetry /usr/local/bin/poetry"
       shell: "bash"
+    - name: "Install Virtualenv compatible with Poetry 1.X"
+      run: |
+        POETRY_VERSION=$(poetry --version | awk '{print $2}')
+        if [[ $(printf "%s\n" "$POETRY_VERSION" "2.0.0" | sort -V | head -n 1) != "$POETRY_VERSION" ]]; then
```

Comment:
Instead of the bash logic here should we just do a github actions conditional on the version input:

```
if: startsWith(inputs.poetry-version, '1.')
```

Reply:
poetry-version is an optional input, I wanted to catch the change even if poetry-version wasn't provided. But now that 2.X is out, I don't know if there would be any ability for poetry 1.x to be installed when poetry-version isn't provided.

Reply:
It defaults to a blank string so this should still work in that case.